src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -526,10 +546,14 @@ public void setDatasetBatchSize(Integer datasetBatchSize) {
     this.datasetBatchSize = datasetBatchSize;
   }
 
-  public Dataset updateDatasetIndex(Integer datasetId, Integer userId, Instant indexDate)
-      throws SQLException {
-    datasetServiceDAO.updateDatasetIndex(datasetId, userId, indexDate);
-    return datasetDAO.findDatasetById(datasetId);
+  private void synchronizeDatasetInESIndex(Dataset dataset, User user) {
```

Comment:
If this was `public`, it could be used in other places where `indexDataset()` is called.